src/main/fc/cli.c

```diff
@@ -350,6 +350,10 @@ static const char * const lookupTableFailsafe[] = {
     "AUTO-LAND", "DROP"
 };
 
+static const char * const lookupTableHorizonTiltMode[] = {
+    "SAFE", "EXPERT"
```

Comment:
In the interest of reducing the amount of flash space that is used, I propose to drop this lookup and rename `horizon_tilt_mode` to `horizon_tilt_expert_mode` and make it into a boolean.